docker/generate_makefile.py

```diff
@@ -58,7 +58,7 @@ def _print_makefile_run_template(image):
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
-            print('\t-entrypoint "/bin/bash" \\\n\t-it ', end='')
+            print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
```

Comment:
I don't like having tabs and newlines everywhere especially given how ugly escaping them is.
I'm not sure that if we didn't have them we could have spotted this, but it's possible.

Reply:
we should actually have a tab only in front of each command of a rule, but not in front of lines after `\`-s. ie.
```
rule:
<tab>command ... \
<spc><spc>continuation
```
we could also remove the `\` newlines altogether

Reply:
+1 to removing newlines completely. this is generated content, does not need to look pretty, and should be less error prone. this newlines and tab removal will be awesome.